modules/ROOT/pages/https-reference.adoc

```diff
@@ -2,9 +2,33 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
+:keywords: configure, HTTPS endpoint
+
+You automatically deploy an API proxy using an HTTP listener, inbound HTTP endpoint, with minimal setup. You can use this approach if you do not foresee any security concerns in your Anypoint Platform API setup. Otherwise, you must configure your API proxy with an HTTPs listener (inbound HTTPS endpoint), instead of an HTTP listener.
```

Comment:
You "can". You not always do that. -> "You can automatically deploy an API proxy"

Reply:
Done! Thanks!